cilium/CFP-32820-improve-scalability-of-health-probing.md

```diff
@@ -0,0 +1,149 @@
+# CFP-32820: Improve Scalability of Health Probing
+
+**SIG: SIG-Scalability**
+
+**Begin Design Discussion:** 2024-08-12
+
+**Cilium Release:** 1.17
+
+**Authors:** Shreya Jayaraman <shjayaraman@microsoft.com> based on the work of Marcel Zięba <marcel.zieba@isovalent.com>
+
+## Summary
+
+Cilium currently recommends disabling ICMP and HTTP health probes for large-scale clusters due to their impacts on cpu/mem usage. We propose to dynamically set the probing interval and thereby provide a configurable number of queries per second issued by probers to reduce the effects of ping frequency at scale.
+
+## Motivation
```

Comment:
I think the motivation section here seems more like background. Maybe that could be renamed.

However, for motivation, I think that the "user journeys" that I highlighted in the original google doc would provide pretty good motivation for the changes:

- "As a cluster administrator, I have a large cluster and want to ensure that the network bandwidth consumed for validating basic connectivity across nodes is well-bounded, even if it means that Cilium can less quickly tell when there is a connectivity breakage on the underlying network."
- "As a cluster administrator, I want to be able to quickly tell whether basic connectivity is OK across nodes, potentially even at the expense of network bandwidth usage within the environment".

Reply:
If we document both of these user journeys as motivations, then I think that helps to motivate why Option 1 (ratio) for configuration can help a wider range of users to tweak this feature to the demands of their environment.

Reply:
This makes a lot of sense to me, thank you for the feedback!